File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.MasterDetail.xUnit._VB/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.vb
'{[{
Imports System.Linq
Imports System.Threading.Tasks
Imports Param_RootNamespace.Core.Services
'}]}

Public class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Fact>
    Public Async Function EnsureSampleDataServiceReturnsMasterDetailDataAsync() As Task
        Dim actual = Await SampleDataService.GetMasterDetailDataAsync()

        Assert.NotEmpty(actual)
    End Function
    '}]}
End Class

File: code/TemplateStudioForUWP/Templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.TreeView.xUnit/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.cs
//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.XUnit
{
    public class Tests
    {
        //^^
        //{[{

        // Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Fact]
        public async Task EnsureSampleDataServiceReturnsTreeViewDataAsync()
        {
            var actual = await SampleDataService.GetTreeViewDataAsync();

            Assert.NotEmpty(actual);
        }
        //}]}
    }
}
